.bazelrc

```diff
@@ -29,3 +29,12 @@ build --http_timeout_scaling=5.0
 # This workaround is due to an incompatibility of
 # bazel_common/tools/maven/pom_file.bzl with Bazel 1.0
 build --incompatible_depset_is_not_iterable=false
+
+# Thread sanitizer configuration:
+build:tsan --strip=never
+build:tsan --copt -fsanitize=thread
+build:tsan --copt -DTHREAD_SANITIZER
+build:tsan --copt -O2
```

Comment:
Did you forget to remove these? Or are `-O2` and `-g` required for the thread sanitizer?

Reply:
The `-g` is to make sure we get all the debug symbols and line numbers in thread sanitizer warnings (which we do) and the -O2 is to make sure it is fast.

Reply:
I'd take out `-O2`, it's really not something that you should override like this. If you want it in Travis or something like that, it should be set in the CI script.

Reply:
Hmm, it is probably better to keep it, because otherwise it might be overridden by the debug flag to be `-O0` or whatever the default for debug builds it.